x/ecocredit/keys.go

```diff
@@ -19,8 +19,8 @@ const (
 
 var ModuleSchema = ormapi.ModuleSchemaDescriptor{
 	SchemaFile: []*ormapi.ModuleSchemaDescriptor_FileEntry{
-		{Id: 1, ProtoFileName: api.File_regen_ecocredit_v1_state_proto.Path()},
-		{Id: 2, ProtoFileName: basketapi.File_regen_ecocredit_basket_v1_state_proto.Path()},
+		{Id: 1, ProtoFileName: basketapi.File_regen_ecocredit_basket_v1_state_proto.Path()},
```

Comment:
changing the ORM table prefix is a breaking change. Reverting the `basket` prefix to `1`.

Reply:
A breaking change is ok for v4.0. Was this causing issues with the migration?

Reply:
Yes. We'll lose all of the basket state.

Reply:
I see. Tested without the change and basket state is empty after the migrations and core state is corrupted including both core state and basket state. Would be nice if we could rename using `Renamed` in `StoreUpgrades` and keep the change here but it looks like that might be more trouble than it's worth.